clang/include/clang/Driver/Options.td

```diff
@@ -2750,6 +2750,11 @@ def fno_global_isel : Flag<["-"], "fno-global-isel">, Group<f_clang_Group>,
   HelpText<"Disables the global instruction selector">;
 def fno_experimental_isel : Flag<["-"], "fno-experimental-isel">, Group<f_clang_Group>,
   Alias<fno_global_isel>;
+def faltmathlib : Joined<["-"], "faltmathlib=">, Group<f_Group>, Flags<[CC1Option]>,
```

Comment:
Should be `faltmathlib_EQ` to be more consistent with a majority of the Joined type options.  Of course, we aren't completely consistent with the options here but we shouldn't introduce new ones that don't use the _EQ.